encoding/codecs/h264.md

```diff
@@ -0,0 +1,247 @@
+# Encoding with x264
+
+H.264 has been the de facto standard video codec
```

Comment:
In line with the explanation in #38, we should differentiate between a codec and a video coding format. H.264 is the format, x264 the codec.